### ISSUE TITLE
Extension menu unavailable and webpages fail to load

### ISSUE THREAD

**Issue by willuwontu** [uBlockOrigin/uBlock-issues#1683]
### Prerequisites

- [X] I verified that this is not a filter issue (MUST be reported at [filter issue tracker](https://github.com/uBlockOrigin/uAssets/issues))
- [X] This is not a [support issue or a question](https://old.reddit.com/r/uBlockOrigin/)
- [X] I performed a [cursory search of the issue tracker](https://github.com/uBlockOrigin/uBlock-issues/issues) to avoid opening a duplicate issue
- [X] The issue is not present after wholly disabling uBlock Origin ("uBO") in the browser
- [X] I checked the [documentation](https://github.com/gorhill/uBlock/wiki) to understand that the issue I report is not a normal behavior

### I tried to reproduce the issue when...

- [X] uBO is the only extension
- [X] uBO with default lists/settings
- [x] using a new, unmodified browser profile

### Description

Webpages attempt to infinitely load while continuing to utilize CPU. The menu that you can normally access by clicking the icon for it is inaccessible. Attempting to load the dashboard from the extensions menu failed to do so.

![image](https://user-images.githubusercontent.com/11213059/128668963-b443b542-44a5-4c9f-af78-9fe67ccb2fc7.png)

While attempting to access local devices, like my wi-fi router, doesn't result in the spinning circle of doom, the page simply fails to load anything until I disable the extension.
![image](https://user-images.githubusercontent.com/11213059/128669044-706d174b-a3a6-487d-9aa6-aa58e22ef656.png)

### A specific URL where the issue occurs

www.google.com

### Steps to Reproduce

1. Install Ublock origin.
2. Go to any web page, including local ones or attempt to access the extension menu.
3. Web page fails to load or menu is unavailable.

### Expected behavior

Webpages to load, ublock origin menu to be accessible.

### Actual behavior

Webpages continually attempted to load, while the menu remained inaccessible.

### uBlock Origin version

1.37.2

### Browser name and version

Google Chrome Version 92.0.4515.131 (Official Build) (64-bit)

### Operating System and version

Windows 10 Version 21Ha (OS Build 19043.1110)

**Comment by uBlock-user** [comment]
For extension menu not appearing --

https://www.reddit.com/r/uBlockOrigin/comments/oq8tmh/ublock_origin_137_announcement_thread/h7ryure/

**Comment by gorhill** [comment]
Unable to reproduce as per repro steps, using Chromium 92.0.4515.107 on Linux Mint.

Whoever can reproduce is welcome to investigate the issue on their site using browser's developer tools. Mind software on your side which may interfere with uBO's installation (i.e. "cleaner", anti-virus, etc.)

**Comment by willuwontu** [comment]
Having done another fresh chrome install and comp restart this morning, it now works for me. I've still got no idea what caused it, but
> Mind software on your side which may interfere with uBO's installation (i.e. "cleaner", anti-virus, etc.), or even a broken Chrome installation.

It was probably something to do with this for me.

**Comment by mopquill** [comment]
A friend of mine asked for help with something that sounds like this same issue on their computer, and I couldn't find the source of the issue. If enabled, web pages will not load, and eventually the plugin crashes.

I then started having the same issue myself on my computer yesterday, and today, Chrome informed me the extension had become corrupt, and I could try to repair it. Doing so appears to have wiped all of my settings and filters (I've got a backup from 2 weeks ago, so I'll be restoring that).

I'm not sure what any of this means, just adding data.

**Comment by gorhill** [comment]
> Chrome informed me the extension had become corrupt

This means Chrome detected the files making up the extension have been found to have changed, and it's not supposed to happen -- this is a browser issue, uBO has no access to your file system and no ability to modify itself, only the browser can do this, or you may have an external application which is corrupting the extension's installation -- for example some "cleaner" or anti-viruses may cause this.

**Comment by mopquill** [comment]
> This means Chrome detected the files making up the extension have been found to have changed, and it's not supposed to happen -- this is a browser issue, uBO has no access to your file system and no ability to modify itself, only the browser can do this, or you may have an external application which is corrupting the extension's installation -- for example some "cleaner" or anti-viruses may cause this.

I understand; I arrived at the same conclusion, however, I found it weird that I got the original error described in this post on two different, unrelated machines (one is my friend's and doesn't sync with this one or anything like that). I came to these issues looking for the very issue being described, having seen two machines with this issue (though my friend's has not [yet?] become corrupt). For what it's worth, I don't use any kind of "cleaner" on Chrome, and my only AV is Windows Defender.

As I said, I am trying to add more data. This is not necessarily a uBlock issue specifically, but something happening due to some recent change that seems to solely affect it. I saw something in my search suggesting Tampermonkey could possibly cause this issue, but I don't know that to be so -- but both my friend and I do indeed have Tampermonkey.
